server/middlewares/validators/activitypub/pagination.ts

```diff
@@ -0,0 +1,23 @@
+import * as express from 'express'
+import { query } from 'express-validator'
+import { logger } from '../../../helpers/logger'
+import { areValidationErrors } from '../utils'
+
+const apPaginationValidator = [
+  query('page').optional().isInt({ min: 1 }).withMessage('Should have a valid page number'),
+  query('size').optional().isInt({ min: 10, max: 50 }).withMessage('Should have a valid page size (min: 10, max: 50)'),
```

Comment:
put them in https://github.com/Chocobozzz/PeerTube/blob/develop/server/initializers/constants.ts#L182

also i don't think a minimum is really useful